shenyu-plugin/shenyu-plugin-api/src/main/java/org/apache/shenyu/plugin/api/result/DefaultShenyuResult.java

```diff
@@ -17,10 +17,14 @@
 
 package org.apache.shenyu.plugin.api.result;
 
+import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlRootElement;
+import org.apache.shenyu.common.constant.Constants;
+
 /**
  * The type Shenyu default result.
  */
-public class DefaultShenyuResult implements ShenyuResult<DefaultShenyuEntity> {
+@JacksonXmlRootElement(localName = Constants.DEFAULT_XML_ROOT)
```

Comment:
default  format xml？

Reply:
default is JSON, there is just a placeholder.